src/pages/index.js

```diff
@@ -125,17 +125,15 @@ class IndexPage extends React.Component {
             <p className="caption text-tertiary">
               {this.props.data.site.siteMetadata.description}
             </p>
-            <a href="#services" className="btn">
-              SEE WORKS
-            </a>
           </div>
           <div className="social-buttons">
             <SocialLinks />
           </div>
         </div>
-        <ServicesList />
-        <BlogList />
-        <Contact />
+        {/* This can be added back as it was originally if needed */}
+        {/* <ServicesList /> */}
+        {/* <BlogList /> */}
+        {/* <Contact /> */}

```

Comment:
I don't want to have this in the home for now. I commented it out so we can add it later if I change my mind.